modules/migrations/github.go

```diff
@@ -97,17 +98,32 @@ func NewGithubDownloaderV3(ctx context.Context, baseURL, userName, password, tok
 			},
 		},
 	}
+
 	if token != "" {
-		ts := oauth2.StaticTokenSource(
-			&oauth2.Token{AccessToken: token},
-		)
-		client = oauth2.NewClient(downloader.ctx, ts)
+		tokens := strings.Split(token, ",")
+		for _, token := range tokens {
+			token = strings.TrimSpace(token)
+
+			ts := oauth2.StaticTokenSource(
+				&oauth2.Token{AccessToken: token},
+			)
+			client = oauth2.NewClient(downloader.ctx, ts)
+
+			downloader.addClient(client, baseURL)
```

Comment:
Is it OK to change `client` here? It was a `http.Client` with proxy and auth settings.

Even the change is OK, a new variable name may make the logic more clear.